src/clr-angular/forms/styles/_checkbox.clarity.scss

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('forms.control.checkbox') {
+    .clr-control-container {
+        .clr-checkbox-wrapper {
```

Comment:
Our rule of thumb is to have CSS selectors nested no more than two deep. We have some that are 3-4 deep but those are the exceptions.

Here `.clr-control-container .clr-checkbox-wrapper input[?]` could just as easily be `.clr-checkbox-wrapper input[?]`.